pysat/_instrument.py

```diff
@@ -212,13 +221,13 @@ def custom_func(inst, opt_param1=False, opt_param2=False):
         inst = pysat.Instrument('pysat', 'testing')
         inst.custom_attach(custom_func, kwargs={'opt_param1': True})
 
-        # Custom methods are applied to data when loaded.
+        # Custom methods are applied to data when loaded
         inst.load(date=date)
 
         print(inst['new_data2'])
 
         # Custom methods may also be attached at instantiation.
-        # Create a dictionary for each custom method and associated inputs
+        # Create a dictionary for each custom method and associated inputs.
```

Comment:
You're being inconsistent with comment punctuation, why add some periods and take away others?

Reply:
The statement I was going from is
```
Block and inline comments should use proper English grammar and punctuation
  with the exception of single sentences in a block, which may then omit the
  final period
```

1. Turns out that isn't in pysat itself. I'm not good at this standards stuff :(
2. The comment wasn't a single sentence.

Reply:
I have a summer school presentation review coming up. I re-read the sentence again. Probably for the best that I take a break.